frontend/app/controllers/TierController.scala

```diff
@@ -61,10 +61,17 @@ object TierController extends Controller with ActivityTracking
       catalog.friend, startDate)).discardingCookies(TierChangeCookies.deletionCookies: _*)
   }
 
-  def change() = SubscriptionAction { implicit request =>
+  def change() = SubscriptionAction.async { implicit request =>
     implicit val countryGroup = UK
     implicit val c = catalog
-    Ok(views.html.tier.change(request.subscriber.subscription.plan.tier, catalog))
+    for {
+      cg <- request.getIdentityCountryGroup
+    } yield {
+      if (cg.exists(_ != CountryGroup.UK) && request.paidOrFreeSubscriber.isLeft)
```

Comment:
`isLeft` doesn't really feel very descriptive here.

Are we using a disjunction with paid details left and free right?